pkg/kubelet/kubeletconfig/configfiles/configfiles.go

```diff
@@ -72,5 +72,51 @@ func (loader *fsLoader) Load() (*kubeletconfig.KubeletConfiguration, error) {
 		return nil, fmt.Errorf("init config file %q was empty, but some parameters are required", path)
 	}
 
-	return utilcodec.DecodeKubeletConfiguration(loader.kubeletCodecs, data)
+	kc, err := utilcodec.DecodeKubeletConfiguration(loader.kubeletCodecs, data)
+	if err != nil {
+		return nil, err
+	}
+
+	// make all paths absolute
+	// skip these, because they have an alternative root location (KubeletFlags.CertDirectory) used to resolve relative paths later on
+	skips := []*string{
```

Comment:
no skip here

Reply:
Ok, SGTM, will do.

Reply:
I finally had some time to really look through the stuff I was concerned about, and I think you're right regarding the resolution behavior - I just hadn't given it a close enough look.